contracts/abstract/Version.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
suggestion: contextual directory naming instead of `abstract`.

Might be inclined to move to the root `contract/` ...as you can probably tell from the contract names, I have a distaste for the popular approach in Solidity of prefix interfaces with 'I' and using directories of `abstract` and `interface` (but that's who other story).

Reply:
moved to `contracts/`